sdcm/nemesis.py

```diff
@@ -4237,15 +4266,16 @@ def _grow_cluster(self, rack=None):
         add_nodes_number = self.tester.params.get('nemesis_add_node_cnt')
         InfoEvent(message=f"Start grow cluster by {add_nodes_number} data nodes").publish()
         new_nodes = []
-        if self.cluster.parallel_node_operations:
-            new_nodes = self.add_new_nodes(count=add_nodes_number, rack=rack,
-                                           instance_type=self.tester.params.get('nemesis_grow_shrink_instance_type'))
-        else:
-            for idx in range(add_nodes_number):
-                # if rack is not specified, round-robin racks to spread nodes evenly
-                rack_idx = rack if rack is not None else idx % self.cluster.racks_count
-                new_nodes += self.add_new_nodes(count=1, rack=rack_idx,
-                                                instance_type=self.tester.params.get('nemesis_grow_shrink_instance_type'))
+        with self._with_action_scope("Grow cluster", target=self.target_node.name, metadata={"grow_nodes_number": add_nodes_number}):
```

Comment:
Does this also write the names of the nodes which were added?

Reply:
added